### DISCUSSION TITLE
Is it possible to pass this question while doing cronjob?

### DISCUSSION THREAD

**Discussion by syofyanzuhad** [stefanzweifel/laravel-backup-restore#18] [Q&A] [answered]
i use this command on schedule, but it won't executed because of this question
```bash
php artisan backup:restore
    --disk=s3
    --backup=latest 
    --connection=mysql 
    --password=my-secret-password 
    --reset
```
![image](https://github.com/stefanzweifel/laravel-backup-restore/assets/52684582/ffdbeaf1-b8db-408c-ba63-1675c86b793a)

**Comment by stefanzweifel** [comment, marked as answer]
Yes, you can use the `--no-interaction` option to prevent the command from asking any questions.

```shell
php artisan backup:restore
    --disk=s3
    --backup=latest 
    --connection=mysql 
    --password=my-secret-password 
    --reset
    --no-interaction
```

Will add a note to the README about this.

**Reply by syofyanzuhad** [reply]
it's work, thankyou
